src/components/templates/PartyMap/components/RoomModal/RoomModal.scss

```diff
@@ -24,18 +24,11 @@ $spacing: 20px;
   &__main {
     display: flex;
     flex-direction: row;
+    flex: auto 1;
     justify-content: space-around;
     align-items: center;
 
-    > img {
-      width: 60%;
-      height: auto;
-
-      border-radius: $border-radius;
-    }
-
     > * {
-      flex: 1;
```

Comment:
What was the intent behind removing this here?

Reply:
Not sure! It actually looks a bit better with it, so I'll put it back

Reply:
The intent of this style is to add flex: 1 to all of the immediate child components, which I believe would tell them to space evenly, and grow/shrink together, etc.

Then I believe we explicitly override the image style elsewhere so it doesn’t actually grow/shrink.